src/components/TransactionHistory/TransactionsGroupByDate.tsx

```diff
@@ -14,7 +14,7 @@ type TransactionsGroupByDateProps = {
 }
 
 type TransactionGroup = {
-  date: number
+  date: string
```

Comment:
👀

Reply:
If you look at the groups now, they are all strings. So had to update the type.

e.g. Today, Yesterday, This Week, etc...

Reply:
Makes sense, would probably make sense to use another vernacular than date (which assumes a unix timestamp), to avoid confusion @reallybeard

Reply:
Yah could prob call it groupName instead...